UnrealEngine.gitignore

```diff
@@ -40,6 +43,7 @@
 *.sdf
 *.VC.db
 *.VC.opendb
+*.code-workspace
```

Comment:
Can we move this out to it's own section and explain why this is needed? If this is being generated by the UE tooling, and it can be ignored, let's add that context. If it's not, then I'm curious why we should special-case it here over what's in `Global/VisualStudioCode.gitignore`

Reply:
That is also generated by Unreal, yes.